app/WsConnection.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TypeFamilies      #-}
+
+{- |Implementation of a connection using Websockets.
+-}
+module WsConnection (WsConnection(..)) where
+
+import           App.ConnectionMgnt
+import           ClassyPrelude
+import qualified Data.Aeson         as Aeson
+import qualified Network.WebSockets as WS
+
+
+newtype WsConnection = WsConnection WS.Connection
+
+instance IsConnection WsConnection where
+    type Pending WsConnection = WS.PendingConnection
+
+    sendMsg (WsConnection conn) =
```

Comment:
why msg? they are plain bytestrings, i think `sendData` is a more fitting name.
For me, Message is more high level.